src/platform/packages/private/kbn-language-documentation/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "@kbn/i18n",
     "@kbn/test-jest-helpers",
     "@kbn/shared-ux-markdown",
+    "@kbn/esql-ast",
```

Comment:
Do we want to avoid this? I need it to add the right types instead of any

Reply:
The CI might complain actually, let's see how it goes

Reply:
So it didnt.  Regardless I would like to avoid it to keep the package as simple as possible I think. 🤔 Mostly because you added them only for the script.

As you added it only for the script I think it is ok to create these interfaces again here and only the paths. You mostly want the license property. I don't have a strong opinion but I think if we can avoid the kbn-ast dependency would be better